UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs

```diff
@@ -1618,7 +1623,8 @@ public void IndicatePain(float dmgTaken)
 		{
 			if (EmoteActionManager.Instance == null || screamEmote == null ||
 			    canScream == false || ConsciousState == ConsciousState.UNCONSCIOUS || IsDead) return;
-			if (dmgTaken >= painScreamDamage) EmoteActionManager.DoEmote(screamEmote, playerScript.gameObject);
+			if (dmgTaken < painScreamDamage) return;
+			EmoteActionManager.DoEmote(screamEmote, playerScript.gameObject);
```

Comment:
by the way just saying this is missing on the Fox Race species thing

Reply:
We don't have meat for foxes

Reply:
As in the scream

Reply:
There are no screams for foxes, so they default to the human screams just like any other species that don't have any screams setup.
Kinda irralavent to this PR.